tsktsk/task.py

```diff
@@ -1,23 +1,37 @@
 import dataclasses
 import textwrap
 from datetime import datetime
+from enum import Enum
 from typing import Optional
 
-CATEGORY_DEFAULT = "NEW"
-VALUE_DEFAULT = "medium"
-EFFORT_DEFAULT = "medium"
 
-CATEGORY = {
-    "NEW": "📦 NEW",
-    "IMP": "👌 IMP",
-    "FIX": "🐛 FIX",
-    "DOC": "📖 DOC",
-    "TST": "✅ TST",
-}
+class Category(Enum):
+    NEW = "📦 NEW"
+    IMP = "👌 IMP"
+    FIX = "🐛 FIX"
+    DOC = "📖 DOC"
+    TST = "✅ TST"
 
-VALUE = {"high": "V⬆", "medium": "", "low": "V⬇"}
-EFFORT = {"high": "E⬆", "medium": "", "low": "E⬇"}
-POINTS = {"high": 8, "medium": 5, "low": 3}
+    DEFAULT = NEW
```

Comment:
Do we want to do this as an attribute? This feels like it makes it an extra value of the enum.

Reply:
It's an alias. It doesn't count as a separate value but it's considered a member.